experimental/rust/image/project/filename.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+bin_name=$(awk '$0 == "[[bin]]" {i=1;next};i && i++ <= 1' Cargo.toml)
+
+echo "$bin_name" | sed -e 's/name = "\(.*\)"/\1/'
```

Comment:
This looks like something that should be within the stack not in the project template